client/lib/plans/constants.js

```diff
@@ -954,9 +954,8 @@ export const FEATURES_LIST = {
 		getTitle: () => i18n.translate( 'VideoPress Support' ),
 		getDescription: () =>
 			i18n.translate(
-				'The easiest way to upload videos to your website and display them ' +
-					'using a fast, unbranded, customizable player with rich stats ' +
-					'(13GB storage space).'
+				'Easy video uploads, and a fast, unbranded, customizable video player, ' +
```

Comment:
It seems like we are not using the feature at the moment. This aside, the copy has been changed to be the same for both 'Unlimited Video Hosting' and 'VideoPress Support' features. Is this intended?

Reply:
Yep, I realize it doesn't show yet, but I changed that too bc VideoPress == Unlimited Video Hosting.